File: app/src/main/java/org/scp/gymlog/room/entities/BarEntity.java
package org.scp.gymlog.room.entities;

import androidx.room.Entity;
import androidx.room.Index;
import androidx.room.PrimaryKey;

@Entity(
        tableName = "bar",
        indices = {
                @Index("weight"),
        }
)
public class BarEntity {
    @PrimaryKey(autoGenerate = true)
    public int barId;
    public int weight;
    public boolean internationalSystem;
}

File: app/src/main/java/org/oar/gymlog/room/entities/BarEntity.kt
package org.oar.gymlog.room.entities

import androidx.room.Entity
import androidx.room.Index
import androidx.room.PrimaryKey

@Entity(
    tableName = "bar",
    indices = [
        Index("weight")
    ]
)
class BarEntity {
    @PrimaryKey(autoGenerate = true)
    var barId = 0
    var weight = 0
    var internationalSystem = false
}